pages/index.vue

```diff
@@ -193,6 +202,8 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@import url('https://fonts.googleapis.com/css2?family=Fira+Code&display=swap');
```

Comment:
nope, no fonts. bad idea.

Reply:
What font would you like me to set the inner code block? monospace?

Reply:
should be good enough, or you can use `Consolas, "Andale Mono WT", "Andale Mono", "Lucida Console", "Lucida Sans Typewriter", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Liberation Mono", "Nimbus Mono L", Monaco, "Courier New", Courier, monospace;` stack

Reply:
Fixed.